scripts/config.py

```diff
@@ -44,6 +43,8 @@ class Config:
     OPT_NVCC = 'NVCC'
     OPT_NVCC_LINKFLAGS = 'NVCC_LINKFLAGS'
     OPT_NVCC_CXXFLAGS = 'NVCC_CXXFLAGS'
+    MINOR_GCC_VERSION = 8
```

Comment:
Minor? Isn't it the major version?
Major versions: 7, 8, 9, 10...
Minor versions: 8.4.2, 8.5.3...
(major = leftmost numbers, minor = rightmost numbers)
Did you mean `MINIMUM_GCC_VERSION` instead?